# HG changeset patch
# User Mike Miller <mtmiller@octave.org>
# Date 1431990881 14400
#      Mon May 18 19:14:41 2015 -0400
# Branch stable
# Node ID b7ee5cefa9d6f45701500e040deb2b439a2aed96
# Parent  f5d9e57d43806797e886fbdf1985ef84ac6b538a
Update make_int example to current octave_base_value API (bug #45136)

* make_int.cc (octave_integer::print): Make non-const.  Add a newline
for consistency with core library conventions.

diff --git a/examples/code/make_int.cc b/examples/code/make_int.cc
--- a/examples/code/make_int.cc
+++ b/examples/code/make_int.cc
@@ -97,31 +97,31 @@ public:
   octave_value transpose (void) const { return new octave_integer (scalar); }
 
   octave_value hermitian (void) const { return new octave_integer (scalar); }
 
   void increment (void) { ++scalar; }
 
   void decrement (void) { --scalar; }
 
-  void print (std::ostream& os, bool pr_as_read_syntax = false) const;
+  void print (std::ostream& os, bool pr_as_read_syntax = false);
 
 private:
 
   int scalar;
 
 
   DECLARE_OV_TYPEID_FUNCTIONS_AND_DATA
 };
 
 void
-octave_integer::print (std::ostream& os, bool pr_as_read_syntax) const
+octave_integer::print (std::ostream& os, bool pr_as_read_syntax)
 {
   os << scalar;
-  // octave_print_internal (os, scalar, pr_as_read_syntax);
+  newline (os);
 }
 
 #ifdef DEFUNOP_OP
 #undef DEFUNOP_OP
 #endif
 
 #define DEFUNOP_OP(name, t, op) \
   UNOPDECL (name, a) \
